runtime.go

```diff
@@ -592,7 +594,18 @@ func parseOutput(w http.ResponseWriter, stdoutTxt io.Reader, runtime string, wg
 		w.WriteHeader(int(statusCode))
 	}
 
-	w.Write([]byte(proxy.Body))
+	// Decode base64 encoded body if isBase64Encoded is true
+	if proxy.IsBase64Encoded {
+		if decodedBytes, err := base64.StdEncoding.DecodeString(string(proxy.Body)); err != nil {
```

Comment:
If we want to be super-precise here in replicating the behavior then we should have two steps. First, we check if the `isBase64Encoded` flag is set. If it is, we decode the Body. Then, we look at the Content-Type returned by the integration (function) and the Accept header sent with the original request, based on these two factors we decide whether to treat the response body as a binary payload (`[]byte`) or a UTF-8 string.

Reply:
Thanks!
I made a change to take Accept header into account.

Reply:
I believe this is the correct behavior. We can go back and make the `acceptMediaTypeMatched` optional later if the service behavior changes